src/components/AppContainer.tsx

```diff
@@ -176,8 +180,6 @@ const AppContainer: React.FC = () => {
       });
 
         client.addListener("tokenExpired", () => {
-          const username = localStorage.getItem("username");
-          const password = localStorage.getItem("password");
           if (username && password) {
             getToken(username, password).then((token) => {
               login(token);
```

Comment:
If no username and password in local storage - should navigate to sign in screen? Client will be stuck unusable here.

Reply:
hm no, they were just moved up to component level

Reply:
https://github.com/twilio/twilio-conversations-demo-react/pull/3/files/70d13de1fb9e9c0f8ca423675f975acc13d1ab26#diff-820ddeb4092719dc94913a9737da496221ee7a2e961440e7dbe18c8c0a8dfdd8R77

Reply:
We went looking everywhere, but couldn’t find those commits.

Reply:
@berkus new link 
https://github.com/twilio/twilio-conversations-demo-react/pull/3/commits/d9484da4e4892b598abc273db7d8da341160d045#diff-820ddeb4092719dc94913a9737da496221ee7a2e961440e7dbe18c8c0a8dfdd8R77

same file `src/components/AppContainer.tsx` 77 line